homeassistant/components/stookwijzer/diagnostics.py

```diff
@@ -4,29 +4,25 @@
 
 from typing import Any
 
-from stookwijzer import Stookwijzer
-
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.core import HomeAssistant
 
-from .const import DOMAIN
-
 
 async def async_get_config_entry_diagnostics(
     hass: HomeAssistant, entry: ConfigEntry
```

Comment:
```suggestion
    hass: HomeAssistant, entry: StookwijzerConfigEntry
```

Reply:
Check, already modified in my local version.